user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,139 @@
+
+As more and more users create their own customized solutions on top of the DataMiner software, we want to allow them to register their solutions to the Catalog as well. This will enable our users to easily manage the solutions and packages they have created, distribute them among the different teams within their organization and even share them with other organizations.
+
+## Private Catalog items
+
+All of the items that have been provided by Skyline Communications are considered to be public items. They are available for all our users. Note that for some of these items (e.g. Connectors) users might need to buy a license to be able to use them on their system.
+
+Private Catalog items are items that are only available to one or more organizations. This gives users the possibility to develop app packages, Automation scripts or any type of Catalog item and redeploy them within their organization on various systems or share them with other organizations.
+
+## Versioning of Catalog items
+
+Catalog items can hold multiple versions. To make sure that the versioning of items is easy to understand for everyone, [Semver](https://semver.org/) versioning is promoted for Catalog item versions. Users can assign extra labels to versions if they want to indicate that a certain version is not an official release (e.g. 1.2.3-alpha).
+
+## How to register a private Catalog item
+
+### Register using the Catalog UI
+
+If you use the UI, you can only register a Catalog item. There is no UI yet to register a new version. This can only be done by using the API.
+
+To register a new Catalog item, do the following:
+
+1. Go to the [Catalog](https://catalog.dataminer.services).
+
+1. Open the user menu by clicking the user icon on the top right. Here you will find the "Register item" button.
+
+1. Indicate for which organization you want to register the Catalog item. This can be done by using the selection box that will automatically be populated with all the organizations on the dataminer.services platform you have access to.
+
+1. Select the type of the Catalog item. The type will define how the Catalog item will be shown to the users. We also allow filtering on type so users can quickly find what they are looking for when browsing the Catalog module.
+
+1. Enter the name of the item. This will be the name that will be used to display the item in the Catalog. It needs to be human readable. Make sure you pick a name that makes sense.
+
+1. Once all information has been filled in, click *Register*. This will conclude the registration process for the Catalog item and return the unique identifier for the item in question. This identifier will allow you to register versions for the catalog item using the [API](xref:Register_Catalog_Item#register-using-the-catalog-api).
+
+### Register using the Catalog API
```

Comment:
perhaps add the b2c authenticated call as well that allows to just register a item (without version). the call that is used by the UI atm.